ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -4990,4 +4991,19 @@ public static boolean arePathsEqualOrWithin(Path p1, Path p2) {
     return ((p1.toString().toLowerCase().indexOf(p2.toString().toLowerCase()) > -1) ||
         (p2.toString().toLowerCase().indexOf(p1.toString().toLowerCase()) > -1)) ? true : false;
   }
+
+  /**
+   * Convenience method to obtain delegation tokens
+   * corresponding to the paths passed for mapReduce job.
+   * @param job jonconf
+   * @param ps array of paths
+   */
+  public static void setToken(JobConf job, Path[] ps) {
+    try {
+      TokenCache.obtainTokensForNamenodes(job.getCredentials(),
+          ps, job);
+    } catch (IOException ex) {
+      LOG.error("Error in setToken ", ex);
```

Comment:
I don't think any errors should be ignored here - if it will not be able to obtain the token; it will not work; or that's not the case?

Reply:
Thank you for your review! 
Agree with you and I have removed the `try catch`